web/ViewModels/PropertyDetailsViewModel.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace BricksAndHearts.ViewModels;
+
+public class PropertyDetailsViewModel
+{
+    public PropertyViewModel Property { get; set; } = null!;

```

Comment:
Theres a bit of a counter pattern here that I am not too happy with, what you are doing here is saying that the Property should not be null (as the type is `PropertyViewModel` not `PropertyViewModel?`), but then immediately set it to null. I think we should either have the property of this class be nullable, or have us set it to a real value to start, probably the former over the latter.